Build.ps1

```diff
@@ -22,8 +22,8 @@ Process {
     
         Exec { & dotnet clean -c Release }
         Exec { & dotnet build -c Release }
```

Comment:
One last thing: the new analyzer project and the associated test project don't declare a Release configuration in the solution file, resulting in this warning below when building on Release configuration. Probably a VS create/add project issue. Removing them/adding them again from the solution should fix the issue.

![image](https://github.com/gsoft-inc/wl-domain-event-propagation/assets/14242083/3e4c546d-5c29-41b8-bcca-90c4a3b20f07)

Reply:
Oh good catch,fixed it.